src/main/java/core/basesyntax/model/ActivitiesType.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.model;
+
+public enum ActivitiesType {
+    b("balance"),
```

Comment:
In Enum we should have full names
```suggestion
    BALANCE("b"),
```

Reply:
It's impossible to link string b with BALANCE then.